streaming/multimodal/convert/webvid/README.md

```diff
@@ -1,13 +1,43 @@
 # WebVid
 
-## crawl.py
+[WebVid](https://m-bain.github.io/webvid-dataset/) is a large-scale dataset of short videos with textual descriptions sourced from stock footage sites. There are two sets of the dataset available:
+1. 10M: It contains 10 million video-text pairs.
+2. 2.5M: It contains 2.5 million video-text pairs.
+
+## Dataset preparation
+
+To use Streaming Dataset, we must first convert the dataset from its native format to MosaicML's Streaming Dataset format called Mosaic Dataset Shard (MDS). Once in MDS format, we can access the dataset from the local file system (disk network attached storage, etc.) or object store (GCS, OCS, S3, etc.). From the object store, data can be streamed to train deep learning models, and it all works efficiently.
+
+Check out the steps below for information on converting WebVid datasets to MDS format—also, checkout [MDSWriter()](https://streaming.docs.mosaicml.com/en/latest/api_reference/generated/streaming.MDSWriter.html) parameters for details on advanced usage.
```

Comment:
Mdash, don't need ()

Reply:
Done in https://github.com/mosaicml/streaming/pull/220/commits/626aa6be526e4e7e11152926881915bb62a9182d.